tests/auctions.test.ts

```diff
@@ -611,15 +627,15 @@ describe('Auctions', () => {
           auctionTxId = writeInteraction.originalTxId;
         });
 
-        it.each([-10, -1, 10, 19, 20, 69])(
```

Comment:
need to look at if we should cap the return value to be the startPrice provided

Reply:
moved these to unit tests and add check in [86e0d0c](https://github.com/ar-io/testnet-contract/pull/126/commits/86e0d0c6072d3334cb6b8867e0ed5834a5c4f3ab)